src/Aux/misc.jl

```diff
@@ -11,12 +11,45 @@ julia> Aux.rotation_matrix_from_axis_angle([1.1, 2.2, 3.3], π/2)
   0.944641    0.285714  0.16131 
  -0.320237    0.695833  0.642857
 ```
-See also: [`Mutators.Dihedral.rotate_dihedral!`](@ref Mutators)
+See also: [`rotate_dihedral!`](@ref Common)
 """
 function rotation_matrix_from_axis_angle(axis::Vector{Float64}, angle::Float64)
     q0 = cos(0.5 * angle)
     q1, q2, q3 = sin(0.5 * angle) * axis ./ norm(axis)
     [1-2*q2*q2-2*q3*q3   2*q1*q2-2*q0*q3   2*q1*q3+2*q0*q2;
        2*q2*q1+2*q0*q3 1-2*q3*q3-2*q1*q1   2*q2*q3-2*q0*q1;
        2*q3*q1-2*q0*q2   2*q3*q2+2*q0*q1 1-2*q1*q1-2*q2*q2]
+end
+
+# -------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+@doc raw"""
+    calc_dih_angle(a1::Vector{Float64}, a2::Vector{Float64}, a3::Vector{Float64}, a4::Vector{Float64}[, radians::Bool = false])
+
+Calculates the dihedral angle produced between a1, a2, a3 and a4, in degrees, by default. If `radians` flag is set to true, the output is in radians.
+
+# Examples
+```julia-repl
+julia> Aux.calc_dih_angle([1.0, 1.0, 1.0], [2.1, 2.1, 2.1], [3.0, 2.0, 5.0], [5.0, 5.0, 5.0], radians = true)
+3.141592653589793
+```
+See also: [`apply_initial_conf!`](@ref Common)
+"""
+function calc_dih_angle(a1::Vector{Float64}, a2::Vector{Float64}, a3::Vector{Float64}, a4::Vector{Float64};
```

Comment:
```suggestion
function calculate_dihedral(a1::Vector{Float64}, a2::Vector{Float64}, a3::Vector{Float64}, a4::Vector{Float64})
```
the function should always return radians